addon/webextension/background/main.js

```diff
@@ -25,6 +26,17 @@ window.main = (function () {
     }
   }
 
+  function setIconActive(active, tabId) {
+    const path = active ? "icons/pageshot-icon-green-38.png" : "icons/pageshot-icon-38.png";
+    browser.browserAction.setIcon({path, tabId});
```

Comment:
Changes the icon to green when the selector is visible. @johngruen I know the color is bad, please change it.